Sources/PaymentButtons/PaymentButtonFont.swift

```diff
@@ -1,12 +1,15 @@
 import UIKit
 
 enum PaymentButtonFont {
-    /// The primary font to be employed in the UI for non special circumstances
-    public static let primaryFont: UIFont = .systemFont(ofSize: UIFont.systemFontSize)
+    /// The primary font
+    public static let paypalPrimaryFont = UIFont(name: "PayPalOpen-Regular", size: 14) ??
+        .systemFont(ofSize: UIFont.systemFontSize)
 
-    /// Usually a sub-heading or alternative text font that is used in some scenarios
-    public static let secondaryFont: UIFont = .systemFont(ofSize: UIFont.smallSystemFontSize)
+    /// The secondary font for sub-heading or alternative text
+    public static let paypalSecondaryFont = UIFont(name: "PayPalOpen-Regular", size: 12) ??
```

Comment:
I'm not sure, but I believe the sources are being added at the application level (Demo App, e.g: Demo/Resources/Fonts....). I think they should be added to PayPal and set as the target membership for PaymentButtons (Sources/PaymentButtons) 🤔

Reply:
Good call, I thought I had the files in `PayPal` but it ended up in `Demo`. Updated thanks!